array_api_compat/dask/array/_aliases.py

```diff
@@ -61,20 +66,24 @@ def astype(
 # This arange func is modified from the common one to
 # not pass stop/step as keyword arguments, which will cause
 # an error with dask
-
-# TODO: delete the xp stuff, it shouldn't be necessary
-def _dask_arange(
+def arange(
     start: Union[int, float],
     /,
     stop: Optional[Union[int, float]] = None,
     step: Union[int, float] = 1,
     *,
-    xp,
     dtype: Optional[Dtype] = None,
     device: Optional[Device] = None,
     **kwargs,
 ) -> Array:
-    _check_device(xp, device)
+    """
+    Array API compatibility wrapper for arange().
+
+    See the corresponding documentation in the array library and/or the array API
+    specification for more details.
+    """
+    # TODO: respect device keyword?
```

Comment:
Worth addressing here, too?

Reply:
Definitely out of scope. `device` on dask is a nuanced mess that can't be solved without proper support by the library itself. Read https://github.com/data-apis/array-api-compat/blob/adbb6efb14c19242ab53c994f943ac6c4d4425f6/array_api_compat/common/_helpers.py#L633-L641